spec/models/coronavirus/sub_section_spec.rb

```diff
@@ -33,5 +33,12 @@
       expect(sub_section).not_to be_valid
       expect(sub_section.errors).to have_key(:content)
     end
+
+    it "fails if featured link is not in content" do
+      sub_section.featured_link = "/bananas"
+
+      expect(sub_section).not_to be_valid
+      expect(sub_section.errors).to have_key(:featured_link)
+    end
```

Comment:
I think we should also have a test for when the featured link is valid, as I don't think that's actually tested at the model level. 

The other two validations in this file seem to be implicitly tested by: https://github.com/alphagov/collections-publisher/blob/6554f8303cc642fde31ca41619580a4dbfee1837/spec/models/coronavirus/sub_section_spec.rb#L17-L21

Reply:
Good shout, I've added this in 🎉